### ISSUE TITLE
"Getting code actions from 'ESLint'" error in VSCode with plugin enabled

### ISSUE THREAD

**Issue by SheldonWBM** [prettier/eslint-plugin-prettier#738]
System:
Ubuntu: 24.04.2 LTS 
VSCode: 1.100.2
yarn: 4.9.1 using dlx sdks
eslint: v9.27.0 (terminal)
"eslint-plugin-prettier": "^5.4.0",
Node: 22.15.1

```workspace
		"typescript.tsdk": "./.yarn/sdks/typescript/lib",
		"typescript.suggest.paths": true,
		"eslint.nodePath": "./.yarn/sdks/eslint", 
```

The issue does not affect the terminal eslint commands, inside or outside Docker containers. 
The issue is VSCode compatibility only.

Issue:
I have a "Getting code actions from 'ESLint'" message when saving a file using VSCode.
In `eslint.config.mjs`, commenting out the following fixes the issue:

```mjs
// import eslintPluginPrettierRecommended from 'eslint-plugin-prettier/recommended';
...
export default tseslint.config(
  {
    ...
  },
  // eslintPluginPrettierRecommended,
  {
  ...
  rules: {
    ...
    // 'prettier/prettier': 'error',
  }
)   
```
No error is displayed in ESLint's output logs (in VSCode).

I have tried changing the `synckit` version in `package.json`, but it does not fix it.
```
"resolutions": {
  ...,
  "synckit": "^0.11.5" // also tried other versions
}
```

I may attempt to downgrade the yarn version, which is a TODO task. For now, commenting out this plugin resolves the issue.

**Comment by JounQin** [comment]
https://github.com/yarnpkg/berry/issues/6335

**Comment by SheldonWBM** [comment]
Fixed, bumped to `eslint-plugin-prettier` to 5.5.1, along with other ESLint-related packages. Fixed itself with time.
